R/shiny.R

```diff
@@ -463,6 +463,36 @@ replaceData = function(proxy, data, ..., resetPaging = TRUE, clearSelection = 'a
   reloadData(proxy, resetPaging, clearSelection)
 }
 
+#' @rdname proxy
+#' @rdname data
+#' 
+#' @export
+updateFilters = function(proxy, data) {
+  # Calculate the values to be supplied to the filters based on column type
+  new_lims = lapply(data, function(x) {
+    if(inherits(x, c('numeric'))) {
```

Comment:
space after `if` (same thing for all other `if`s in both R and JS code):

```suggestion
    if (inherits(x, c('numeric'))) {
```